tests/unit/test_network_connections.py

```diff
@@ -2998,6 +2998,106 @@ def test_valid_persistent_state(self):
         input_connection.update({"state": "absent"})
         self.assertValidationError(validator, input_connection)
 
+    def test_set_deprecated_master(self):
+        """
+        When passing the deprecated "master" it is updated to "controller".
+        """
+        input_connections = [
+            {
+                "name": "prod2",
+                "state": "up",
+                "type": "bridge",
+                "interface_name": "bridge2",
```

Comment:
interface_name can be omitted, can't it?

Reply:
I think yes, let me try.

Reply:
Fixed!